### PR TITLE
Greatly extend library functionality

### PR DESCRIPTION
**Pull request by antal-bukos**
- change string formatting to be compatible with python 2.7
- tweak logic for generating list and dict types
- export enums
- allow lists for context, for cases where a type is needed in multiple places

### CODE REVIEW

File: typemallow/__init__.py [hunk 1]
```diff
@@ -36,21 +36,28 @@ def __get_ts_interface(schema):
     data type.
 
     '''
```
Comment (gnagel) [inline typemallow/__init__.py:38]: Why do we not need to replace the schema substring here?

File: typemallow/__init__.py [hunk 2]
```diff
@@ -36,21 +36,28 @@ def __get_ts_interface(schema):
...
+            ts_type = value.nested.__name__
             if value.many:
                 ts_type += '[]'
+        if type(value) is fields.List:
+            item_type = value.container.__class__
+            if item_type is fields.Nested:
+                # TODO: figure out type of nested list item
+                ts_type = 'unknown[]'
+            else:
+                ts_type = '{}[]'.format(mappings.get(item_type, 'unknown'))
```
Comment (gnagel) [inline typemallow/__init__.py:52]: Why unknown here?

File: typemallow/__init__.py (hunk 1 above)
Comment (antal-bukos) [reply typemallow/__init__.py:38]: We don’t strip the word from the flow types either, I was thinking we would want to keep the same names for the types.

File: typemallow/__init__.py (hunk 2 above)
Comment (antal-bukos) [reply typemallow/__init__.py:52]: The linter doesn’t allow ‘any’, I’ve seen ‘unknown’ being used by other FE devs.

**Comment by antal-bukos** [comment]
This was moved into the main repo, closing.

**Pull request closed without merging**